roles/test_operator/tasks/run-test-operator-job.yml

```diff
@@ -67,6 +67,11 @@
     - not cifmw_test_operator_dry_run | bool
     - not testjob_timed_out
   block:
+    - name: Initialize volumes and volume_mounts
```

Comment:
You are actually reseting the values right? Maybe the task name could say that. Not a blocking to merge